### PR TITLE
add single color support

### PR DESCRIPTION
**Pull request by NathanaelA**
This adds code to:
- Detects if being ran as `root`, and errors out if not.
- Auto detect if using `system76/` or `system76_apci/` directory.   The newer devices are using `system76_acpi/` instead of `system76/` as the path to the leds.
- Detect if the keyboard uses 3 color zones or a single color zone and then hides the extra buttons if they are not needed and uses the correct path for either `color` or `color_center` depending if it has zones or not.

Basically this PR fixes the issue for the newer driver and new devices (Tested on Bonobo) while retaining compatibility for the older devices and older drivers.

### CODE REVIEW

**Comment by ahoneybun** [comment]
LGTM thank you for the work there! I'd love to work together more on this project. I can confirm it works on the oryp4 as well.
